docs/content/mobile/android/local-deploy.md

```diff
@@ -1,73 +1,65 @@
-[//]: # (TODO - get rid of mobile_launch.json mention and explain it differently)
-
-# Connecting to a local deployment
+# Connect to a local deployment
 
 Before reading this document, make sure you are familiar with:
 
-  * [Setting up Android Support for the GDK]({{urlRoot}}/content/mobile/android/setup)
-  * [Ways to test your Android client]({{urlRoot}}/content/mobile/android/ways-to-test)
-
-## Connecting your Android device to a local deployment using Unity Remote
-You need the Unity Remote app for this. See the [Unity documentation](https://docs.unity3d.com/Manual/UnityRemote5.html) for details.
+* [Setting up Android Support for the GDK]({{urlRoot}}/content/mobile/android/setup)
+* [Ways to test your Android client]({{urlRoot}}/content/mobile/android/ways-to-test)
 
-  1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
-  1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
-  1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
-  1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
-  1. In the **Unity Remote** section, click on the drop-down menu beside the **Device** option and select **Any Android Device**.
-  1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
-  1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
+## Prepare your project to connect to a local deployment
 
-    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+To connect your Android device to a local deployment, you need to prepare; follow the steps below to ensure any local deployment you launch via the Unity Editor is correctly set up.
```

Comment:
`To connect your Android device to a local deployment, you need to prepare;` -> what do you need to prepare?